developer-hub/src/reference_docs/extrinsic_encoding.rs

```diff
@@ -0,0 +1,156 @@
+//! # Constructing and Signing Extrinsics
+//!
+//! Substrate is configurable enough that extrinsics can take any format, in theory. In practice,
+//! runtimes tend to use our [`sp_runtime::generic::UncheckedExtrinsic`] type to represent extrinsics.
+//! In Polkadot, this is configured
+//! [here](https://github.com/polkadot-fellows/runtimes/blob/94b2798b69ba6779764e20a50f056e48db78ebef/relay/polkadot/src/lib.rs#L1478)
```

Comment:
All in all you should give a disclaimer before going forward everything that follows in this page is only valid if you stick to this type. If you craft your own extrinsic type, things might differ. 

But we should also denote that `UncheckedExtrinsic` is generic AF so that hopefully everyone can be happy while using it, so that individual teams rarely need to touch these super sensitive parts of the code.